src/Sarif.Multitool/MultitoolResources.resx

```diff
@@ -129,6 +129,21 @@
   <data name="ErrorNoGroupInFileName" xml:space="preserve">
     <value>Error: No "group" (the part of the file name before the dash) found in '{0}'. All file names must be of the form [Group]-[RunID] and each adjacent pair in the same group is compared.</value>
   </data>
+  <data name="ErrorUnknownFilingTargetType" xml:space="preserve">
+    <value>Error: The value of the '{0}' option is not a known filing target type.
```

Comment:
>Error: The value of the '{0}' option is not a known filing target type. [](start = 11, length = 71)

(ryley says) should we show the wrong value that user inputted in these messages for faster context?

Reply:
yes we should. I don't like this Error: prefix convention, what's that all about? 

---
In reply to: [306569098](https://github.com/microsoft/sarif-sdk/pull/1583#discussion_r306569098) [](ancestors = 306569098)

Reply:
Messages modified to mention incorrect value; "Error:" removed.

---
In reply to: [306578099](https://github.com/microsoft/sarif-sdk/pull/1583#discussion_r306578099) [](ancestors = 306578099,306569098)